### PR TITLE
Moved guidelines.. .

### PR DESCRIPTION
**Pull request by waldo1001**
Started to move guidelines from C/AL to AL as described in: https://github.com/microsoft/alguidelines/discussions/63

### CODE REVIEW

**Comment by waldo1001** [comment]
Too soon .. will move all guidelines first, then make the pullrequest.

**Comment by waldo1001** [comment]
For the remaining guidelines to be copied - I'll create indivitual PRs.

File: content/BCBestPractices/Readability/begin-as-an-afterword/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
+
+## Good code
+
+```al
+    if ICPartnerRefType = ICPartnerRefType::"Common Item No." then begin
+    ...
+    end;
+```
+
```
Comment (TheDoubleH) [inline content/BCBestPractices/Readability/begin-as-an-afterword/index.md:28]: How do we handle multi-line conditions?

```AL
if (ICPartnerRefType = ICPartnerRefType::"Common Item No.") and
   (ICPartnerRefType::"Common Item No." <> '')
then begin
   blah
end;
```

or

```AL
if (ICPartnerRefType = ICPartnerRefType::"Common Item No.") and
   (ICPartnerRefType::"Common Item No." <> '') then begin
     ...
     blah
end;
```

Comment (waldo1001) [reply content/BCBestPractices/Readability/begin-as-an-afterword/index.md:28]: I'd say:

```al
if (ICPartnerRefType = ICPartnerRefType::"Common Item No.") and
   (ICPartnerRefType::"Common Item No." <> '') then begin
     ...
     blah
end;
```

Comment (waldo1001) [reply content/BCBestPractices/Readability/begin-as-an-afterword/index.md:28]: In Fact - there is a guideline about this - and I'm wrong :D - I'll attach the PR.

Comment (waldo1001) [reply content/BCBestPractices/Readability/begin-as-an-afterword/index.md:28]: Here you go: https://github.com/microsoft/alguidelines/pull/75

**Review by JesperSchulz — APPROVED** [review]

**Review by ajkauffmann — APPROVED** [review]

**Review by TheDoubleH — APPROVED** [review]

**Pull request merged**